plugins/splunk-on-call/src/components/EntitySplunkOnCallCard.tsx

```diff
@@ -108,10 +151,38 @@ export const EntitySplunkOnCallCard = () => {
       {},
     );
     const teams = await api.getTeams();
-    const foundTeam = teams.find(teamValue => teamValue.name === team);
-    return { usersHashMap, foundTeam };
+    let foundTeams = [
+      teams.find(teamValue => teamValue.name === teamAnnotation),
+    ].filter(team => team !== undefined);
```

Comment:
☝️ Note that `foundTeams` (previously `foundTeam`) is now an array. In instances where a `splunk.com/on-call-team` annotation is used (rather than `splunk.com/on-call-routing-key`), `foundTeams.length` should be `1` and therefore the UX remains the same for those users utilizing a `splunk.com/on-call-team`.